src/components/templates/FireBarrel/FireSeat.tsx

```diff
@@ -1,130 +1,43 @@
-      if (!user || !!token) return;
-
-      const getToken = firebase.functions().httpsCallable("video-getToken");
-      const response = await getToken({
-        identity: user.uid,
-        room: roomName,
-      });
-
-      setToken(response.data.token);
-    })();
-  }, [user, roomName, token]);
-
-  const [room, setRoom] = useState<Video.Room>();
-  const [videoError, setVideoError] = useState<string>("");
-
-  useEffect(() => {
-    return () => {
-      if (room && room.localParticipant.state === "connected") {
-        room.localParticipant.tracks.forEach(function (trackPublication) {
-          //@ts-ignored
-          trackPublication.track.stop(); //@debt typing does this work?
-        });
-        room.disconnect();
-      }
-    };
-  }, [room]);
-
-  const connectToVideoRoom = () => {
-    if (!token) return;
-    setVideoError("");
-
-    Video.connect(token, { name: roomName })
-      .then((room) => setRoom(room))
-      .catch((error) => setVideoError(error.message));
-  };
-
-  const [participants, setParticipants] = useState<Array<Video.Participant>>(
-    []
-  );
-  useEffect(() => {
-    if (!token) return;
-    let localRoom: Video.Room;
-    const participantConnected = (participant: Video.Participant) => {
-      setParticipants((prevParticipants) => [
-        // Hopefully prevents duplicate users in the participant list
-        ...prevParticipants.filter((p) => p.identity !== participant.identity),
-        participant,
-      ]);
-    };
-
-    const participantDisconnected = (participant: Video.Participant) => {
-      setParticipants((prevParticipants) => {
-        if (!prevParticipants.find((p) => p === participant)) {
-          // Remove when root issue foudn and fixed
-          console.error("Could not find disconnnected participant:");
-          console.error(participant);
-        }
-        return prevParticipants.filter((p) => p !== participant);
-      });
-    };
-
-    Video.connect(token, { name: roomName })
-      .then((room) => {
-        setRoom(room);
-        localRoom = room;
-        room.on("participantConnected", participantConnected);
-        room.on("participantDisconnected", participantDisconnected);
-        room.participants.forEach(participantConnected);
-      })
-      .catch((error) => {
-        setVideoError(error.message);
-      });
-
-    return () => {
-      if (localRoom && localRoom.localParticipant.state === "connected") {
-        localRoom.localParticipant.tracks.forEach((trackPublication) => {
-          // @ts-ignored
-          trackPublication.track.stop();
-        });
-        localRoom.disconnect();
-      }
-    };
-  }, [participants, token, roomName]);
-
-  const users = useSelector((state) => state.firestore.data.partygoers);
-  const profileData = room ? users[room.localParticipant.identity] : undefined;
-
+// TODO: potentially don't even need this anymore, can delete maybe if you want? I inlined the only relevant looking bits from here
```

Comment:
Calling this out, after ripping out all the bits and pieces there didn't even seem to be much left in here.